diff --git a/node_modules/remark-parse/lib/tokenize/strong.js b/node_modules/remark-parse/lib/tokenize/strong.js
index 62d5524..0753716 100644
--- a/node_modules/remark-parse/lib/tokenize/strong.js
+++ b/node_modules/remark-parse/lib/tokenize/strong.js
@@ -42,35 +42,51 @@ function strong(eat, value, silent) {
     return
   }
 
+  // this may begin a combined emphasis/strong, in which case we need to ensure the strong is the outer style before proceeding
+  var strongEmph = value.charAt(index) === marker;
+  if (strongEmph) {
+    index++;
+    queue = marker;
+  }
+
   while (index < length) {
     previous = character
     character = value.charAt(index)
 
     if (
       character === marker &&
-      value.charAt(index + 1) === marker &&
       (!pedantic || !whitespace(previous))
     ) {
-      character = value.charAt(index + 2)
+      if (value.charAt(index + 1) !== marker && strongEmph)
+        strongEmph = false;
+      if (value.charAt(index + 1) === marker) {
+        if (value.charAt(index + 2) === marker) {
+          queue += marker;
+          index++;
+          strongEmph = false;
+        } else if (strongEmph) return; // need to let emphasis eat first
 
-      if (character !== marker) {
-        if (!trim(queue)) {
-          return
-        }
+        character = value.charAt(index + 2)
 
-        /* istanbul ignore if - never used (yet) */
-        if (silent) {
-          return true
-        }
+        if (character !== marker) {
+          if (!trim(queue)) {
+            return
+          }
+
+          /* istanbul ignore if - never used (yet) */
+          if (silent) {
+            return true
+          }
 
-        now = eat.now()
-        now.column += 2
-        now.offset += 2
+          now = eat.now()
+          now.column += 2
+          now.offset += 2
 
-        return eat(subvalue + queue + subvalue)({
-          type: 'strong',
-          children: self.tokenizeInline(queue, now)
-        })
+          return eat(subvalue + queue + subvalue)({
+            type: 'strong',
+            children: self.tokenizeInline(queue, now)
+          })
+        }
       }
     }
 
