diff --git a/node_modules/widdershins/lib/openapi3.js b/node_modules/widdershins/lib/openapi3.js
index d673b05..c032fc3 100644
--- a/node_modules/widdershins/lib/openapi3.js
+++ b/node_modules/widdershins/lib/openapi3.js
@@ -637,7 +637,7 @@ function convertInner(api, options) {
         data.translations = {};
         templates.translations(data);
 
-        data.version = (data.api.info && data.api.info.version && typeof data.api.info.version === 'string' && data.api.info.version.toLowerCase().startsWith('v') ? data.api.info.version : 'v' + (data.api.info ? data.api.info.version : 'v1.0.0'));
+        data.version = (data.api.info && data.api.info.version && typeof data.api.info.version === 'string' && data.api.info.version.toLowerCase().startsWith('v') ? data.api.info.version : 'v' + (data.api.info && data.api.info.version ? data.api.info.version : '1.0.0'));
 
         let header = {};
         header.title = api.info && (api.info.title || 'API') + ' ' + data.version;
