diff --git a/node_modules/@parcel/reporter-dev-server/lib/ServerReporter.js b/node_modules/@parcel/reporter-dev-server/lib/ServerReporter.js
index 67e0f2e..0804d41 100644
--- a/node_modules/@parcel/reporter-dev-server/lib/ServerReporter.js
+++ b/node_modules/@parcel/reporter-dev-server/lib/ServerReporter.js
@@ -16,7 +16,7 @@ var $WaiqN$crypto = require("crypto");
 var $WaiqN$events = require("events");
 var $WaiqN$parcelplugin = require("@parcel/plugin");
 var $WaiqN$parcelutils = require("@parcel/utils");
-var $WaiqN$punycode = require("punycode");
+var $WaiqN$punycode = require("punycode/");
 var $WaiqN$child_process = require("child_process");
 
 function $parcel$export(e, n, v, s) {
