diff --git a/node_modules/gatsby-plugin-mdx/utils/gen-mdx.js b/node_modules/gatsby-plugin-mdx/utils/gen-mdx.js
index e558745..9f79611 100644
--- a/node_modules/gatsby-plugin-mdx/utils/gen-mdx.js
+++ b/node_modules/gatsby-plugin-mdx/utils/gen-mdx.js
@@ -104,7 +104,7 @@ export const _frontmatter = ${JSON.stringify(data)}`
   // in the future it'd be nice to not do this twice
   debug(`generating AST`)
   const compiler = mdx.createMdxAstCompiler(options)
-  results.mdast = compiler.parse(content)
+  results.mdast = compiler.parse({ contents: content, path: node.fileAbsolutePath })
 
   /* await mutateNode({
    *   pluginOptions,
